### ISSUE TITLE
Feature Request: Allow specifying option description without short name

### ISSUE THREAD

**Issue by yfakariya** [Cysharp/ConsoleAppFramework#51]
### Background

Currently, help message is generated from `[Option]` attribute's positional argument. However, for relatively minor options, it is hard to assign short name because they are only 26 chars. So, it is useful to enable specifying help message text without short name.

### Proposal

* Add support for `[System.ComponentModel.Description]` attribute for command parameters.
* If both of `[Option]` with description and `[Description]` are specified, `[Description]` to improve sense of unity when options with `[Option]` and options with `[Description]` are mixed by specifying all descriptions are specified through `[Description]` attributes.

### Example

```c#
public void Foo(
  [Option("i")][Description("Specify input file path.")] string input,
  [Option("o")][Description("Specify output file path.")] string output,
  [Description("Allows overwrite output file.")] bool overwrite = false
)
{
   ...
}
```

**Comment by neuecc** [comment]
Thank you.
I would like to include the feature, but I don't really like the idea of adding more attributes to use.
How about accepting null and `[Option(null, "description")]`?
It's not smart, though.

**Comment by yfakariya** [comment]
Thank you for the proposal. I think `[Option(null, "...")]` is OK. It looks easy for developers to find because I had been tried `[Option("", "...")]` (and got a bizarre help message).

I would also propose second option that just using named argument (settable property). It allows more properties to be added in the future (for example, specifying name which cannot use C# identifier like `my-long-option`).

**Comment by neuecc** [comment]
yes, but `[Option(***, "description", Description = "")]` was very strange at IntelliSense view...

**Comment by neuecc** [comment]
released `3.1.0`.
